tests/dx11.rs

```diff
@@ -54,9 +53,6 @@ fn test_imgui_dx11() {
         }
     }
 
-    TermLogger::init(LevelFilter::Trace, Config::default(), TerminalMode::Mixed, ColorChoice::Auto)
-        .unwrap();
-
```

Comment:
Can we add a tracing subscriber here? Seeing logs is pretty important for tests (same for other tests).